slick/src/main/scala/akka/stream/alpakka/slick/javadsl/Slick.scala

```diff
@@ -64,6 +68,23 @@ object Slick {
   ): Flow[T, java.lang.Integer, NotUsed] =
     flow(session, 1, toStatement)
 
+  /**
+   * Java API: creates a Flow that takes a stream of elements of
+   *           type T, transforms each element to a SQL statement
+   *           using the specified function, and then executes
+   *           those statements against the specified Slick database.
+   *
+   * @param session The database session to use.
+   * @param toStatement A function that creeates the SQL statement to
```

Comment:
```suggestion
   * @param toStatement A function that creates the SQL statement to
```
(same error on the method above BTW)

Reply:
OMG. I fixed this in other places, but itself copy&past incorrect version 🤦

Reply:
✅ Done